### PR TITLE
:recycle: Move Creditor and refactor closeMargin

### PR DESCRIPTION
**Pull request by zekiblue**

### CODE REVIEW

**Review by Thomas-Smets — APPROVED** [review]
gtg

File: src/accounts/AccountV1.sol [hunk 1]
```diff
@@ -520,8 +520,8 @@ contract AccountV1 is AccountStorageV1, IAccount {
     }
 
     /**
-     * @notice Calls external action handler to execute and interact with external logic.
-     * @param actionHandler The address of the action handler.
+     * @notice Calls external Action Multicall to execute and interact with external logic.
```
Comment (Thomas-Smets) [inline src/accounts/AccountV1.sol:523]: Guess this has to be aligned with changes in Arcadia Lending?

Comment (j-vp) [reply src/accounts/AccountV1.sol:523]: already in Lending dev!

File: src/interfaces/ICreditor.sol [hunk 2]
```diff
@@ -4,20 +4,32 @@
...
+ * @title Creditor implementation.
+ * @author Pragma Labs
+ * @notice This contract contains the minimum functionality a Creditor, interacting with Arcadia Accounts, needs to implement.
+ * @dev For the implementation of Arcadia Accounts, see: https://github.com/arcadia-finance/accounts-v2.
+ */
 interface ICreditor {
     /**
-     * @notice Checks if account fulfills all requirements and returns application settings.
-     * @param accountVersion The current version of the Account.
+     * @notice Checks if Account fulfils all requirements and returns Creditor parameters.
```
Comment (j-vp) [inline src/interfaces/ICreditor.sol:15]:
```suggestion
     * @notice Checks if Account fulfills all requirements and returns Creditor parameters.
```

(American English)

**Pull request merged**